src/Validator.php

```diff
@@ -112,7 +112,6 @@ public function validate(
             } else {
                 /** @psalm-suppress MixedAssignment */
                 $validatedData = $dataSet->getAttributeValue($attribute);
-                $context->setParameter(ValidationContext::PARAMETER_VALUE_AS_ARRAY, null);
```

Comment:
It is need.

Reply:
It's not necessary because of the usage of `ArrayHelper` with default value in `getParameter()`:

https://github.com/yiisoft/validator/blob/4d0c9407118f76a641908fcadd19b5557ceab381/src/ValidationContext.php#L275

Reply:
Fixed it differently.